### ISSUE TITLE
Question:  Can you suggest a more appropriate forum for a new adopter to post questions?

### ISSUE THREAD

**Issue by trainor1** [django-helpdesk/django-helpdesk#1094]
Hi All,

This is NOT a bug report.  _Bug Report_ was just the least bad option available to categorize this issue.

In a nutshell, I am looking for another forum where I can post questions about Django-helpdesk and get answers from the community.  This doesn't seem like the appropriate place for my questions about installation problems and such.

Provided that I can get Django-helpdesk working and adopt it, I am interested in being a contributor in the long term.  But, right now, I need a place where folks can help me get unstuck from beginner issues.  I am hoping that you can suggest the right place.

Thanks in advance for your help.

Best Regards,

Kevin

**Comment by uhurusurfa** [comment]
This is a good place and there are plenty of people willing to help you get started.
Post your questions and someone is sure to help.

**Comment by Benbb96** [comment]
Maybe the [discussion tab](https://github.com/django-helpdesk/django-helpdesk/discussions) could be more appropriate for threads like this ? I have notifications enabled on this project so I may help if I see something I can answer 😉

**Comment by uhurusurfa** [comment]
👍 That works

**Comment by trainor1** [comment]
Thanks for the replies.  Are you guys sure that I should try Discussions?  I didn't see any content over there.  I think that is what brought me here.  If someone is going to be listening, I would be glad to be the first to post.

Kevin

**Comment by Benbb96** [comment]
Indeed, I often see these warnings and I always think we should fix it. First easy solutions would be to delete these two commands as they doesn't seem to be used... Or we take time to update the way the command options are added.

I'm looking on this page to understand how I can do it : https://docs.python.org/3/library/argparse.html#the-add-argument-method

**Comment by Benbb96** [comment]
@trainor1 I started a PR in that direction : #1095

**Comment by uhurusurfa** [comment]
Commands updated by #1213
